models/brats_mri_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,64 @@
+
+# Brats MRI 3D Latent Diffusion Generative Model
+
+This model is a generator for creating images like the Flair MRIs based on BraTS 2018 data.. It was trained as a 3d latent diffusion model and accepts Gaussian random noise as inputs to produce an image output. The `train_autoencoder.json` file describes the training process of the variational autoencoder with GAN loss, and is based on the [MONAI GAN tutorials](https://github.com/Project-MONAI/tutorials/blob/main/modules/mednist_GAN_workflow_dict.ipynb).
+
+This is a demonstration network meant to just show the training process for this sort of network with MONAI.
+
+### Downloading the Dataset
+The training data is from the Multimodal Brain Tumor Segmentation Challenge (BraTS) 2018.
```

Comment:
link of brats 2018 dataset can use:
[Multimodal Brain Tumor Segmentation Challenge (BraTS) 2018](https://www.med.upenn.edu/sbia/brats2018.html)

Reply:
Thank you!